src/model/package.ts

```diff
@@ -21,7 +21,7 @@ export type QueryOptions = object & BaseQueryOptions;
 
 // A regexp that matches paths to external code.
 // TODO(rictic): Make this extensible (polymer.json?).
-const MATCHES_EXTERNAL = /(^|\/)(bower_components|node_modules)/;
+const MATCHES_EXTERNAL = /(^|\/)(bower_components|node_modules|build($|\/))/;
```

Comment:
I don't think this Regexp is right...  I think it should be `/(^|\/)(bower_components|node_modules|build)($|\/)/`

We don't *need* to add a test for it but it wouldn't hurt...